README.md

```diff
@@ -23,350 +23,46 @@
 <img src="assets/github-hr.png" alt="---" />
 <br/>
 
-**k6** is a modern load testing tool, building on [our](https://k6.io/about) years of experience in the load and performance testing industry. It provides a clean, approachable scripting API, [local](https://k6.io/docs/getting-started/running-k6) and [cloud execution](https://k6.io/docs/cloud), and flexible configuration.
+**k6** is a modern load-testing tool, building on [our years of experience](https://k6.io/about) in the performance and testing industries.
+k6 is built to be simple, powerful, and extensible. Every commit to this repo aims to help create an application with **high performance and excellent developer experience**.
+Some key features include:
 
-This is how load testing should look in the 21st century.
+- **Configurable load generation.** Even lower-end machines can simulate lots of traffic.
+- **Tests as code.** Reuse scripts, modularize logic, version control, and integrate tests with your CI.
+- **A full-featured API.** The scripting API is packed with features that help you simulate real application traffic.
+- **An embedded JavaScript engine.** The performance of Go, the scripting simplicity of vanilla ECMAscript (without baggage from the wider JS ecosystem).
+- **Multiple Protocol support**. HTTP, WebSockets, gRPC, and more.
+- **Large extension ecosystem.** You can extend k6 to support your needs. And many people have already shared their extensions with the community!
```

Comment:
Maybe let's link to the Extensions docs page here?

Reply:
To stay consistent, I'd like to leave this section linkless, but I should add a link and bullet for extensions in the next section :+1: